<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advancedSetupDialog</class>
 <widget class="QDialog" name="advancedSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>204</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>273</width>
    <height>204</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>273</width>
    <height>204</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Advanced Setup</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/simModel.png</normaloff>:/simModel.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>183</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Transducer Setup</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>232</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="backingCheckBox">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="text">
        <string>PZT Backing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="windowedSourceCheckBox">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="text">
        <string>Windowed Source</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="waveSourceComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Longitudinal Wave Source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Shear Wave Source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Longitudinal and Shear Wave Sources</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="sourceTypeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Pressure</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Displacement</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Presure &amp; Displacement</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>advancedSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>advancedSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
